kong/tracing/propagation/schema.lua

```diff
@@ -0,0 +1,61 @@
+local Schema = require "kong.db.schema"
+local utils = require "kong.tools.utils"
+local formats = require "kong.tracing.propagation.utils".FORMATS
+
+
+local extractors = {}
+for _, ext in pairs(formats) do
```

Comment:
Could we iterator over the directory instead so we can simply add a Lua file to support a new format?

Reply:
that thought crossed my mind too, but I don't think we need to make the schema "too smart", better go for a clear and reliable approach where formats are statically defined as constants in my opinion. We don't add new formats often enough to justify something like this, but let me know if you disagree, it is certainly feasible.